galaxy/wrapper/plotHeatmap.xml

```diff
@@ -134,6 +140,10 @@
             <when value="yes">
                 <expand macro="sortRegions" />
                 <expand macro="sortUsing" />
+                <param argument="--sortUsingSamples" type="text" size="30"
+                    label="List of samples to be used for sorting"
+                    help="List of sample numbers (order as in matrix), which are used by --sortUsing for sorting.
+                    If no value is set, it uses all samples. Example: 1 3 (space separated!)">
```

Comment:
Can you please add:

```
            <sanitizer invalid_char="">
                <valid initial="string.digits">
                    <add value=" "/>
                </valid>
            </sanitizer>
```

This will make sure only digits and space and be entered.